Resources/Locale/en-US/disposal/unit/components/disposal-unit-component.ftl

```diff
@@ -1,11 +1,13 @@
+disposal-unit-no-hands = You don't have hands!
```

Comment:
i am absolutely positive this exact Loc exists somewhere else. Regardless, not having hands is not disposal specific and should probably be placed somewhere else (general interaction loc?)

Reply:
Nice im changin this to interaction-popup-component